forge-ai/src/main/java/forge/ai/ComputerUtilCombat.java

```diff
@@ -2480,6 +2480,8 @@ public static int predictPoisonFromTriggers(Card attacker, Player attacked, int
                 }
                 // TODO: Predict replacement effects for counters (doubled, reduced, additional counters, etc.)
             }
+            // intern toxic effect
+            poison += attacker.getKeywordMagnitude(Keyword.TOXIC);
```

Comment:
`hasDoubleStrike()` should probably be moved to the bottom so it only gets done once and also counts for Toxic

But I can do that later if you want